wcc-contentful/lib/wcc/contentful/events.rb

```diff
@@ -32,12 +32,10 @@ def _attach_listeners
       WCC::Contentful::Services.instance.sync_engine
     ]
 
-    if defined?(Rails)
-      publishers << WCC::Contentful::WebhookController
-    end
-    
+    publishers << WCC::Contentful::WebhookController if defined?(Rails)
+
     publishers.each do |publisher|
-      publisher.subscribe(self, with: :rebroadcast)
+      publisher.subscribe(self, with: :rebroadcast) if publisher.present?
```

Comment:
when do you have something in `publishers` that is `nil`?

Reply:
SyncEngine can be nil if the store does not respond to `:index?`.